apps/astarte_realm_management/test/astarte_realm_management/queries_test.exs

```diff
@@ -258,6 +258,60 @@ defmodule Astarte.RealmManagement.QueriesTest do
     end)
   end
 
+  test "get_interfaces_details/1 returns interface with mappings" do
+    {:ok, _} = DatabaseTestHelper.connect_to_test_database()
+    client = connect_to_test_realm("autotestrealm")
+
+    json_obj =
+      Jason.decode!("""
```

Comment:
If you use a valid interface object here instead of decoding a JSON, you can just skip the validation (lines 292-299) and go straight up to building the automaton and installing the interface

Reply:
Changed.